gym/wrappers/monitoring.py

```diff
@@ -0,0 +1,41 @@
+from gym import monitoring
+from gym import Wrapper
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+def Monitor(directory, video_callable=None, force=False, resume=False,
```

Comment:
Agreed, have a fix here to move away from the decorator syntax. Will merge in the morning:
https://github.com/openai/gym/pull/442